src/AppBundle/Model/Event.php

```diff
@@ -78,6 +79,8 @@ class Event
         'file-usage',
         'items-created',
         'items-improved',
+        // TODO: remove this
```

Comment:
Why is it a `TODO` to get rid of this?

Reply:
This is list of metrics that will be shown in UI. Displaying them is helpful when debugging, but not needed in production.